docs/source/format/Integration.rst

```diff
@@ -455,14 +455,16 @@ Gold File Integration Tests
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Pre-generated json and arrow IPC files (both file and stream format) exist
-in the `arrow-testing <https://github.com/apache/arrow-testing>` repository
+in the `arrow-testing <https://github.com/apache/arrow-testing>`__ repository
```

Comment:
Do we need two `_`s here?

```suggestion
in the `arrow-testing <https://github.com/apache/arrow-testing>`_ repository
```

Reply:
Strictly speaking the double underscore is "more" correct if this link name (arrow-testing) isn't reused, so I have the habit of writing the double underscore for links like this (https://stackoverflow.com/questions/27420317/restructured-text-rst-http-links-underscore-vs-use). But in practice it doesn't matter really, both work fine in most cases.

Reply:
Wow! I didn't know it. Thanks!